README.md

```diff
@@ -1,38 +1,31 @@
 # Developer Similarity
 
-This is a legacy from source{d}'s research in developer similarity once it was a [recruitment business](http://web.archive.org/web/20161020170426/http://sourced.tech/).
+This is a legacy from [source{d}](http://www.sourced.tech/)'s research in developer similarity when its main goal was to [match developers and companies](http://web.archive.org/web/20161020170426/http://sourced.tech/).
 We were obsessed with developer embeddings and recommending similar ones. Not everything we created
 is included here. For now, it's the profile images to visually compare the experience.
 
-source{d} had (and is going to have again by the end of 2017) a nearly complete mirror of GitHub repositories.
-Everything related to working with developer metadata requires understanding which email addresses corerespond
-to which GitHub users. GitHub ToU directly forbids to use API for recruitment purposes which we clearly had,
-so we saw no easy way to obtain that information. Luckily, we were able to perform identity matching using
-various heuristics for Git commit signatures. For example, given two different signatures with the same name
-in the same repository, it is likely that the different email addresses belong to the same user. We never
-measured the accuracy of our heuristics - back then we were a humble startup - but our CTO claims that
-it worked reasonably well in practice.
+source{d} runs a massive-scale data retrieval pipeline that among other sources nearly mirrors all GitHub repositories.
```

Comment:
which at ~the~ **that** time? Is this a misprint or a feature? :)

Reply:
Feature. Both work.